src/tests/ftest/server/cpu_usage.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from apricot import TestWithServers
+from general_utils import run_task
+import time
+
+
+class CPUUsage(TestWithServers):
+    """Test Class Description:
+    Measure CPU usage of daos_server with target = 16 and verify that it's
+    less than 100%.
+    :avocado: recursive
+    """
+
+    def test_cpu_usage(self):
+        # pylint: disable=pylint-bad-continuation
+        """
+        JIRA ID: DAOS-4826
+        Test Description: Test CPU usage of daos_server.
+        :avocado: tags=all,server,small,full_regression,cpu_usage
```

Comment:
Seeing `Test-tag-hw-small: pr,hw,small cpu_usage` it seems like your intent is to use hardware cluster. Am I understanding it right?
If so, you might want to add `hw` tag in the tag list.

Reply:
Done. Thanks